docs/source/AdministratorGuide/ServerInstallations/HTTPSServices.rst

```diff
@@ -0,0 +1,76 @@
+.. _https_services:
+
+==================================
+Installing HTTPs services in DIRAC
+==================================
+
+More and more DIRAC services are available through the HTTPs protocol instead of the historical DISET (``dips://``) protocol.
+
+
+General principle
+=================
+
+Contrary to the DISET protocol where each service would have its own process and port opened, HTTPs services are served by a unique process and port, based on Tornado.
+
+There are exceptions to that, due to the use of global variables in some parts of DIRAC. Namely:
+
+* Master CS
```

Comment:
```suggestion
* Master Configuration Service
```
Plus add link to https://dirac.readthedocs.io/en/latest/AdministratorGuide/Systems/Configuration/